tests/python/route_planning/test_custom_order.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import fields2cover as f2c
+
+PARAMETRIZED_INPUTS = [
+  (5, [0, 1, 2, 3, 4], [0, 1, 2, 3, 4]),
+  (10, range(10), range(10))
+]
+
+PARAMETRIZED_INPUTS_WRONG = [
+  (5, [0, 1, 2, 3, 4], [1, 2, 3, 4, 5]),
+  (5, [0, 1, 2, 3, 4], [1, 1, 2, 2, 4]),
+  (10, range(10), range(1,11))
+]
+
+@pytest.mark.parametrize('n,custom_order,expected', PARAMETRIZED_INPUTS)
+def test_fields2cover_custom_order(n, custom_order, expected):
+  swaths = f2c.Swaths()
+  for i in range(n):
+    swaths.push_back(
+        f2c.Swath(
+          f2c.LineString(
+            f2c.VectorPoint([
+              f2c.Point(0, i),
+              f2c.Point(1, i)])), 
+          1, 
+          i
+        )
+    )
+  swath_order = f2c.RP_CustomOrder(swaths, f2c.VecSize(custom_order))
+  swaths = swath_order.genSortedSwaths()
+  actual = [swath.getId() for swath in swaths]
+  assert len(actual) == len(expected)
+  assert all(a == b for a, b in zip(actual, expected))
+
+@pytest.mark.parametrize('n,custom_order,expected', PARAMETRIZED_INPUTS_WRONG)
+def test_fields2cover_wrong_custom_order(n, custom_order, expected):
+  swaths = f2c.Swaths()
+  for i in range(n):
+    swaths.push_back(
+        f2c.Swath(
+            f2c.LineString(
+                f2c.VectorPoint([f2c.Point(0, i),
+                                 f2c.Point(1, i)])), 1, i))
+  swath_order = f2c.RP_CustomOrder(swaths, f2c.VecSize(custom_order))
```

Comment:
pytest error. VecSize doesn't exist. Do you mean VectorSize?

Reply:
Yes, similar as above